packages/pwa/app/components/_app-config/index.jsx

```diff
@@ -90,8 +118,16 @@ AppConfig.restore = (locals = {}) => {
     // Parse the locale from the page url.
     const locale = getLocale(locals) || DEFAULT_LOCALE
     const currency = getPreferredCurrency(locale) || DEFAULT_CURRENCY
-
-    locals.api = new CommerceAPI({...apiConfig, locale, currency})
+    const agent = getAgent
+
+    locals.api = new CommerceAPI({
+        ...apiConfig,
+        locale,
+        currency,
+        fetchOptions: {
```

Comment:
I'd prefer if we only set `fetchOptions` server-side.

Although setting this today _should_ have no effect client-side, I'm not sure that will always be true.

Reply:
That argument is good enough for me to make a change. Thanks.